crypto/s2n_rsa.h

```diff
@@ -15,31 +15,35 @@
 
 #pragma once
 
-#include <openssl/rsa.h>
 #include <stdint.h>
 
-#include "stuffer/s2n_stuffer.h"
+#include <openssl/rsa.h>
 
 #include "crypto/s2n_hash.h"
 
 #include "utils/s2n_blob.h"
 
-struct s2n_rsa_public_key {
-    RSA *rsa;
-};
+struct s2n_pkey;
 
-struct s2n_rsa_private_key {
+struct s2n_rsa_key {
     RSA *rsa;
```

Comment:
I think the original intent was to match the openssl naming scheme. I've found a lot of code that looks something like this:
```
struct s2n_rsa_key rsa_key;
...
rsa_key.rsa
```
For these instances, it looks cleaner to me to keep it as `rsa`.